### DISCUSSION TITLE
missionpinball.org/latest/  links not resolving propperly.

### DISCUSSION THREAD

**Discussion by DorusvdLinden** [missionpinball/missionpinball#155] [Q&A]
I automatically always get the /latest/ added to the https://missionpinball.org/ link:
https://missionpinball.org/latest/

When clicking on the "Beginner's guide to editing the docs" link below any page it tries to go to:
missionpinball.org/about/help

which does not resolves into:
missionpinball.org/latest/about/help
but gives me an error 404 warning

I tried in both edge & chrome.
Can someone reproduce this?

(link to issue: https://github.com/missionpinball/mpf-docs/issues/538)

**Comment by worldpeace-germany** [comment]
can confirm that I see the same behavior.
